assets/js/components/KeyMetrics/KeyMetricsSetupCTAWidget.js

```diff
@@ -113,14 +114,14 @@ function KeyMetricsSetupCTAWidget( { Widget, WidgetNull } ) {
 		);
 	}, [ viewContext ] );
 
-	const [ hasBeenInView, setHasBeenInView ] = useState( false );
-
-	useEffect( () => {
-		if ( isTooltipVisible && ! hasBeenInView ) {
+	useMount( () => {
+		// Since components are conditionally rendered, when tooltip
+		// appears, old component will unmount and new componnet will mount,
+		// with tooltip visible equal to true, so here we ensure event is sent only once when that occurs,
```

Comment:
Watch out for the typos :) We should fix these, and probably try to improve the clarity of overall comment in a subsequent PR...

```suggestion
		// appears, old component will unmount and new component will mount,
		// with tooltip visible equal to true, so here we ensure event is sent only once when that occurs.
```